### PR TITLE
Update Docs for CSCC API

### PR DESCRIPTION
**Pull request by blueandgold**

### CODE REVIEW

File: _docs/_latest/configure/notifier/index.md [hunk 1]
```diff
@@ -171,6 +171,36 @@ notifier:
             webhook_url: https://hooks.slack.com/services/foobar
 ```
 
+### CSCC Notification
+
+Forseti Security can configured to send violations to [Cloud Security Command Center (Cloud SCC)](https://cloud.google.com/security-command-center/).
+
+1. Open `forseti-security/configs/server/forseti_conf_server.yaml`.
```
Comment (joecheuk) [inline _docs/_latest/configure/notifier/index.md]: We removed the "/server/" folder in the GCS bucket and locally for inconvenience.

File: _docs/_latest/configure/notifier/index.md [hunk 2]
```diff
@@ -171,6 +171,36 @@ notifier:
...
+The options below are available for you to configure:
+
+* `enabled:`
+  * **Description**: Whether to send notification to Cloud SCC.
+  * **Valid values**: one of valid `true` or `false`
+
+* `mode:`
+  * **Description**: How to send the violations to Cloud SCC.
+  * **Valid values**: one of valid `api` or `bucket`
+  * **Note**: `api` mode will only work if Forseti is using a service account
```
Comment (joecheuk) [inline _docs/_latest/configure/notifier/index.md]: Is there any external doc we can link to that tells the user how to get a service account from CSCC?

File: _docs/_latest/configure/notifier/index.md [hunk 3]
```diff
@@ -171,6 +171,36 @@ notifier:
...
+from the Cloud Security Center project.
+
+* `organization_id:`
+  * **Description**: The organization id.
```
Comment (joecheuk) [inline _docs/_latest/configure/notifier/index.md]: Can we also add something like "used only when model is api" and same for the gcs bucket description.

File: _docs/_latest/configure/notifier/index.md (hunk 1 above)
Comment (blueandgold) [reply _docs/_latest/configure/notifier/index.md]: Done.

File: _docs/_latest/configure/notifier/index.md (hunk 2 above)
Comment (blueandgold) [reply _docs/_latest/configure/notifier/index.md]: Good idea, done.

File: _docs/_latest/configure/notifier/index.md (hunk 3 above)
Comment (blueandgold) [reply _docs/_latest/configure/notifier/index.md]: Done

**Comment by blueandgold** [comment]
Closing this...something made changes to 500+ files

**Pull request closed without merging**